tests/dash/packets.py

```diff
@@ -71,7 +71,7 @@ def get_pl_overlay_dip(orig_dip, ol_dip, ol_mask):
     return str(ip_address(overlay_dip))
 
 
-def inbound_pl_packets(config, inner_packet_type='udp', vxlan_udp_dport=4789):
+def inbound_pl_packets(config, is_smartswitch=False, inner_packet_type='udp', vxlan_udp_dport=4789):
```

Comment:
Suggest renaming `is_smartswitch` to `use_pkt_alt_attrs` or similar to match here: https://github.com/sonic-net/sonic-mgmt/pull/17324/files#r2023793417